### PR TITLE
fixed struct 'Ts' literal to 'Timestamp' of type models.Attachment

### PR DESCRIPTION
**Pull request by mradspieler**
Hi miraclesu, the rocket.chat SDK made a change in models.Attachment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"log"
 	"time"
 
-	"github.com/miraclesu/rocketrus"
+	"github.com/mradspieler/rocketrus"
```
Comment (miraclesu) [inline README.md]: why change the import path?

File: rocketrus.go [hunk 2]
```diff
@@ -198,10 +198,10 @@ func (rh *RocketrusHook) Fire(e *logrus.Entry) error {
 	}
 
 	msg := &models.Attachment{
-		Color: color,
-		Title: e.Level.String() + " log",
-		Ts:    e.Time.String(),
-		Text:  e.Message,
+		Color:     color,
+		Title:     e.Level.String() + " log",
+		Timestamp: e.Time.String(),
```
Comment (miraclesu) [inline rocketrus.go:203]: where is the doc?
I didn't see any change.
https://github.com/RocketChat/Rocket.Chat.Go.SDK/blob/master/models/message.go#L47
https://rocket.chat/docs/developer-guides/rest-api/chat/postmessage/#attachments-detail

File: README.md (hunk 1 above)
Comment (mradspieler) [reply README.md]: sorry my mistake, i revert this change

File: rocketrus.go (hunk 2 above)
Comment (mradspieler) [reply rocketrus.go:203]: you changed the field Ts in the SDK https://github.com/RocketChat/Rocket.Chat.Go.SDK/commit/2a98370dd0f119b89235fb7bcdec2a4f748ca4a4#diff-c2de9728f932398a66758c56ed794296  to Timestamp but not in rocketrus.go

**Comment by mradspieler** [comment]
Sorry I will close that pull request an send a new one with the correct commit

**Pull request closed without merging**